tutorials/3d/merge_groups.rst

```diff
@@ -0,0 +1,248 @@
+grandchildren of the steering wheel (unless one of them explicitly reactivates
+merging with an ``On`` ``merge_mode``).
+
+
+Ways to use MergeGroups
+-----------------------
+
+There are three ways to use MergeGroups:
+
+AutoMerge
+^^^^^^^^^
+
+The ``MergeGroup`` will attempt to merge any descendent merges as it enters the
+``SceneTree``.
+This is the simplest method, and the best introduction to merging.
+
+Baking in the Editor
+^^^^^^^^^^^^^^^^^^^^
+
+When the ``MergeGroup`` is selected in the editor, a new ``bake`` button should
+appear in the toolbar. This allows you to bake the entire merged scene out to a
+file (.tscn (text scene) or .scn (binary scene)) at *design time* rather than at
+*runtime*. This can be used creatively to build sensible composite objects which
+are later used to compose a game level (this general approach is often known as
+"kitbashing").
+
+Additionally, baking in advance offers a great way to preview what will happen
+when merging at runtime. It allows you to open the merged scene, and see which
+meshes were successfully merged, and which were problematic.
+
+.. figure:: img/bake_merge_group.webp
+    :align: center
+
+Pros
+~~~~
+
+-  Allows you easily see the results of merging
+-  Allows you to further build levels using these merged scenes
+-  No time is taken by the merging process at runtime
+
+Cons
+~~~~
+
+-  Merged scenes typically require substantially more storage than the original meshes
+-  The size of your exported game will likely increase
+-  Larger merged scenes may take longer to load
+-  Larger merged scenes may take up more RAM at runtime, especially on the GPU
+
+If you merge, e.g. 10 boxes, the merged scene will have to store 10 times as
+much geometry data, as the polygons are duplicated. More storage means your
+exported game will be larger, the data will take longer to load, and consume
+more RAM at runtime. For this reason, baking in advance tends to be more
+practical with low poly meshes and art styles.
+
+.. tip:: Due to the increased storage requirements, it is recommended that, wherever possible, you bake scenes in binary format (.scn) rather than text (.tscn). This is because binary scenes are much more compact in terms of storage.
+
+Manually at runtime
+^^^^^^^^^^^^^^^^^^^
+
+If the ``automerge`` property of a ``MergeGroup`` is disabled, then the node
+will do nothing at runtime until you call its ``merge_meshes()`` function.
+Triggering merging manually in this way has two major use cases:
+
+1) Procedural levels. If you place objects (e.g. trees, boxes) at runtime using
+script, rather than placing them at design time, then you want a way to *delay*
+merging until after your placement is complete. Manually calling
+``merge_meshes()`` allows this.
+
+2) Merging parameters. These can be set via the ``MergeGroup::set_param()`` and
+``MergeGroup::set_param_enabled()`` functions, prior to calling
+``merge_meshes()``.
```

Comment:
```suggestion
1. Procedural levels. If you place objects (e.g. trees, boxes) at runtime using
   script, rather than placing them at design time, then you want a way to *delay*
   merging until after your placement is complete. Manually calling
   ``merge_meshes()`` allows this.
2. Merging parameters. These can be set via the ``MergeGroup::set_param()`` and
   ``MergeGroup::set_param_enabled()`` functions, prior to calling
   ``merge_meshes()``.
```